### ISSUE TITLE
Change Password Bug

### ISSUE THREAD

**Issue by jasperkelder** [CriticalMoments/CMSaasStarter#13]
When signing up in the demo with an email the change password feature does not seem to work. Based on the feedback given by the site it wrongly identified my account as an OAuth account and thus didn't allow changing the password. Using the forgot password feature did work.

**Comment by scosman** [comment]
Working fine for me. Can you provide some detail: steps, screenshot of error, ideally email address on account (can DM to me if you want).

<img width="611" alt="Screenshot 2024-02-10 at 2 20 39 PM" src="https://github.com/CriticalMoments/CMSaasStarter/assets/848343/bad72451-207e-40f9-aa83-66955af99c14">

**Comment by jasperkelder** [comment]
Yes sure. So pretty straight forward, I did a sign up with an email address and a randomly generated password (length 16), clicked the confirm email link I got send, went to settings and clicked change password. I'll DM you the address I used, initially I deleted the account afterwards, but I redid the process just now. The supplied image is the message shown in /account/settings/change_password

![Screenshot 2024-02-11 at 09 57 29](https://github.com/CriticalMoments/CMSaasStarter/assets/66300715/bcd68817-691d-447b-a841-f4ffff94a75b)
